### PR TITLE
【微扩展】环境变量bug 修复 #5023

### PR DESCRIPTION
**Pull request by yjieliang**
【微扩展】环境变量bug 修复 #5023

### CODE REVIEW

File: src/backend/ci/core/store/api-store/src/main/kotlin/com/tencent/devops/store/api/common/UserStoreEnvVarResource.kt [hunk 1]
```diff
@@ -105,7 +125,7 @@ interface UserStoreEnvVarResource {
         storeCode: String,
         @ApiParam("生效范围 TEST：测试 PRD：正式 ALL：所有，用\",\"分隔进行拼接", required = false)
         @QueryParam("scopes")
-        @BkField(patternStyle = BkStyleEnum.SCOPE_STYLE, required = false)
+        @BkField(patternStyle = BkStyleEnum.SCOPE_STYLE, required = true)
         scopes: String?,
```
Comment (carlyin0801) [inline src/backend/ci/core/store/api-store/src/main/kotlin/com/tencent/devops/store/api/common/UserStoreEnvVarResource.kt:129]: 这个字段是非必填的，BkField注解的required应该为false

**Review by carlyin0801 — APPROVED** [review]
ok

File: src/backend/ci/core/store/biz-store/src/main/kotlin/com/tencent/devops/store/service/common/impl/StoreEnvVarServiceImpl.kt [hunk 2]
```diff
@@ -148,6 +261,8 @@ class StoreEnvVarServiceImpl @Autowired constructor(
     ): Result<List<StoreEnvVarInfo>?> {
         logger.info("storeEnvVar getLatestEnvVarList userId:$userId,storeType:$storeType,storeCode:$storeCode")
         val storeTypeObj = StoreTypeEnum.valueOf(storeType)
+        val scopeList = if (scopes.isNullOrBlank()) "ALL,PRD,TEST".split(",")
+        else scopes.split(",")
```
Comment (carlyin0801) [inline src/backend/ci/core/store/biz-store/src/main/kotlin/com/tencent/devops/store/service/common/impl/StoreEnvVarServiceImpl.kt:265]: "ALL,PRD,TEST".split(",")为什么要这么写，初始化一个数组不是更好

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=6357) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-ci?pullRequest=6357) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: yjieliang<br/>:x: v_yjieliang<br/><hr/>**v_yjieliang** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-ci?pullRequest=6357) it.</sub>

**Pull request closed without merging**